### DISCUSSION TITLE
How can we build a version that supports both JWT and has an optional non-JWT version for non-embedded apps?

### DISCUSSION THREAD

**Discussion by ncpope** [osiset/laravel-shopify#928] [Ideas]
Hi all, I would like to start a discussion on the best path forward to create a non-embedded app using v17 and above.

I would like to see understand what steps would be required to have this as an option for apps that can't be embedded or those that would be greatly impacted by migrating to an embedded version.

The goal is to outline what steps would be needed and if this is in fact still a possibility with this package.

Hopefully, it would lead to a possible pull request so that non-embedded apps and embedded apps could both make use of this package. I am happy to work on this feature but I would like some initial input from you all first.

**Comment by gnikyt** [comment]
I added some notes in the #929 issue. I'm all for adding it back in and can assist. I think leaving the issue open as a feature request is fine.

**Comment by ncpope** [comment]
@osiset sounds good, I don't think I can delete this discussion, I don't see an option on my end. Could you delete it for me as I think the issue is ideal for this. Thanks!
